File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06PD1/StrategyWkitoria/Strategy/SportsmenCommandMultiply.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Strategy
{
    internal class SportsmenCommandMultiply : SportsmenCommandComplex
    {
        private uint count;
        private SportsmenCommand cmd;
        public SportsmenCommandMultiply(string name, uint count, SportsmenCommand cmd) : base(name)
        {
            this.count = count;
            this.cmd = cmd;
        }
        public override string execute(Sportsmen sportsmen)
        {
            return count.ToString()+" * "+cmd.execute(sportsmen);
        }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2021s/IZ08IO1/Command/src/SportsmenCommandMultiply.java

public class SportsmenCommandMultiply extends SportsmenCommandComplex
{
   private SportsmenCommand cmd;
   private int count;
   public SportsmenCommandMultiply(String name,SportsmenCommand cmd,int count)
   {
      super(name);
      this.cmd=cmd;
      this.count=count;
   }
   @Override public String execute(Sportsmen sportsmen)
   {
      return String.format("%d x %s",count,cmd.execute(sportsmen));
   }
}
